### PR TITLE
[BHE] fix parameters' documentation

### PR DESCRIPTION
**Pull request by ChaofanChen**
As titled.

### CODE REVIEW

File: Documentation/ProjectFile/prj/processes/process/HEAT_TRANSPORT_BHE/borehole_heat_exchangers/borehole_heat_exchanger/t_bhe_if_use_python_bc.md [hunk 1]
```diff
@@ -0,0 +1 @@
+It provides users with freedom of whether enabling the pipe network among the multiple BHEs or not. If there is no corresponding key word, the default setting is 'false'.
```
Comment (endJunction) [inline Documentation/ProjectFile/prj/processes/process/HEAT_TRANSPORT_BHE/borehole_heat_exchangers/borehole_heat_exchanger/t_bhe_if_use_python_bc.md]: drop prosa, typo in keyword

```suggestion
Enables the pipe network among the multiple BHEs. If there is no corresponding key word, the default setting is 'false'.
```

File: ProcessLib/HeatTransportBHE/BHE/CreateBHECoaxial.cpp [hunk 2]
```diff
@@ -34,11 +34,11 @@ parseBHECoaxialConfig(
 {
     // if the BHE is using python boundary condition
     auto const bhe_if_use_python_bc_conf =
+        //! \ogs_file_param{prj__processes__process__HEAT_TRANSPORT_BHE__borehole_heat_exchangers__borehole_heat_exchanger__bhe_if_use_python_bc}
         config.getConfigParameter<bool>("bhe_if_use_python_bc", false);
```
Comment (endJunction) [inline ProcessLib/HeatTransportBHE/BHE/CreateBHECoaxial.cpp]: The keyword itself could also be improved, as mentioned in the original PR.
Something like `enable_bhe_pipe_network`, or `use_bhe_pipe_network`.

File: Documentation/ProjectFile/prj/processes/process/HEAT_TRANSPORT_BHE/borehole_heat_exchangers/borehole_heat_exchanger/t_bhe_if_use_python_bc.md (hunk 1 above)
Comment (ChaofanChen) [reply Documentation/ProjectFile/prj/processes/process/HEAT_TRANSPORT_BHE/borehole_heat_exchangers/borehole_heat_exchanger/t_bhe_if_use_python_bc.md]: Thanks, changed.

File: ProcessLib/HeatTransportBHE/BHE/CreateBHECoaxial.cpp (hunk 2 above)
Comment (ChaofanChen) [reply ProcessLib/HeatTransportBHE/BHE/CreateBHECoaxial.cpp]: Yes, the keyword has been changed to 'use_bhe_pipe_network' after discussing with Shuang Chen.

**Pull request merged**